### PR TITLE
Removes magicarps from xenobio

### PR DESCRIPTION
**Pull request by Archanial**
## About The Pull Request

Title

also removed commented out stuff that was unused for long time

## Why It's Good For The Game

They are unbalanced and not good for the game

## Changelog
:cl:
del: Removed magicarps from xenobio spawn.
/:cl:

### CODE REVIEW

**Comment by RDS88-beeman** [comment]
im pretty sure this is an ided PR after he got beaten a blob round because of a revive carp

assuming that blob was him which i think it was

**Comment by BitBite112** [comment]
Someone's salty.

**Comment by Archanial** [comment]
> assuming that blob was him which i think it was

I wasn't the blob

**Comment by BitBite112** [comment]
Carp is very underutilized, there's no need to remove it unless it gets spammed every single round. This was a one time occurrence.

**Comment by Archanial** [comment]
> 
> 
> Carp is very underutilized, there's no need to remove it unless it gets spammed every single round. This was a one time occurrence.

Well I very obviously disagree and you don't provide any arguments that justify keeping them

**Comment by RDS88-beeman** [comment]
keep them because it's fun?
actually is it fun?
i'm gonna go with it is fun
but i don't spam golds so
also didn't we already delete changebolt?

**Comment by Archanial** [comment]
They still have instant death
They still have resurrection 
They still have fireball

They are only fun for the one making them. You can't freely shoot with them because that's grief, and they are not fun for anyone that is on receiving end of INSTANT FUCKING DEATH.
Also they just make whole medbay worthless with their resurrection.

**Comment by PowerfulBacon** [comment]
cant you just uncomment the xenobiology subtype?

**Comment by RDS88-beeman** [comment]
alright alright but are you aware
are you aware gold slimes delete
"Gold will also require a re-do of the basic Gold effects at the same time as the crossbreed merges:

Inject core with Water to get up to 50u of milk
Inject core with Blood to get up to 5 sheets of leather (10u blood each)
Inject core with at least 5u Plasma to get a random simplemob butcher"

**Comment by BitBite112** [comment]
Instant death got removed from the magicarp pools as well as bolt of change rather recently.
Even  if something is insanely powerful, it should only be removed once it becomes a problem. This hasn't been spammed and overutilized so hasn't become a problem.
And so what if they make medbay useless? It's not even true. You still need chemistry for bombs and healing during a combat situation. Magicarps can't do surgery either.

File: code/modules/events/wizard/magicarp.dm [hunk 1]
```diff
@@ -45,12 +45,6 @@
 	projectiletype = pick(allowed_projectile_types)
 	. = ..()
 
-/*/mob/living/simple_animal/hostile/carp/ranged/xenobio
-	desc = "45% magic, 50% carp, 5% slime, 100% horrible."
-	allowed_projectile_types = list( /obj/item/projectile/magic/animate, /obj/item/projectile/magic/teleport, /obj/item/projectile/magic/door, /obj/item/projectile/magic/aoe/fireball,
-	/obj/item/projectile/magic/spellblade, /obj/item/projectile/magic/arcane_barrage)
-	gold_core_spawnable = HOSTILE_SPAWN*/
-
```
Comment (PowerfulBacon) [inline code/modules/events/wizard/magicarp.dm:53]: this tbh

Comment (Archanial) [reply code/modules/events/wizard/magicarp.dm:53]: teleporting, fireballing, creating doors, just what the game needs

**Comment by Rukofamicom** [comment]
This is already going to happen when the upcoming gold crossbreeds merge happens (mostly). 

Dominant Gold / Rainbow will be the only way to get any carp and it'll be the Chaos Magicarp with no reliable output. Your spell might heal the target, or might completely explode in your face. All the fun of magic fish with non of the reliability of a specific spell every couple of seconds.

**Comment by Effemu** [comment]
i thought you were based 😔 
dont remove magicarps they are cool

**Comment by MNarath1** [comment]
xenobio just had the strongest carp move removed from all i would be down for changing the probability of carps trough so that other mobs get favored more making rnd rolls for a good roll harder

**Comment by MNarath1** [comment]
Or you know include some of the more crappy bolt types into it reducing the likelyhood of a good roll more trough thats hardly possible cause magicarp already has blade energy probably the weakest magic bolt of all tbh

File: code/modules/events/wizard/magicarp.dm (hunk 1 above)
Comment (PowerfulBacon) [reply code/modules/events/wizard/magicarp.dm:53]: not the ability to create doors

**Comment by FortSumter** [comment]
I'm of the opinion the entire xenbiology department is bad for the game. That being said, I'm also against removing fun. Dont remove carp, just the spells it can cast. Remove polymorph and death bolt and its balanced. Revive is balanced to me because its exceptionally rare and it's always good to have things that keep players from being permanently removed from the game.

**Comment by MNarath1** [comment]
> I'm of the opinion the entire xenbiology department is bad for the game. That being said, I'm also against removing fun. Dont remove carp, just the spells it can cast. Remove polymorph and death bolt and its balanced. Revive is balanced to me because its exceptionally rare and it's always good to have things that keep players from being permanently removed from the game.

Then its not a magicarp anymore but just a regular space carp
Also polymorph already got removed just a bit ago in a PR.

**Comment by FortSumter** [comment]
A carp that shoots fireballs or heals people is still better than a normal carp and better than no magikarp at all.

**Comment by zeskorion** [comment]
> cant you just uncomment the xenobiology subtype?

^^^ just do this, its more likely to get in anyways. you need to remove the default one from the pool though

**Comment by Glcbitj** [comment]
Yeah bro sure, let's remove one of the few useful AND FUN gold slime mobs amongst the sea of useless crap it spawns

**Review by utilbird — CHANGES_REQUESTED** [review]
What bacon & zesko said

**Comment by Archanial** [comment]
> 
> 
> What bacon & zesko said

no

**Pull request closed without merging**